docs/src/pages/nextjs/pagedir.mdx

```diff
@@ -89,20 +88,26 @@ const handler = createNextPageApiHandler({
 export default handler;
 ```
 
-### Creating The UploadThing Components (optional)
+### Creating The UploadThing Components (recommended)
 
-Generating components let's you pass your generic `FileRouter` once and then
-have typesafe components everywhere, instead of having to pass the generic
-everytime you mount a component, but you can also import the components
+Generating components lets you pass your generic `FileRouter` and `url` once and
+then have typesafe components everywhere, instead of having to pass the generic
+and url everytime you mount a component, but you can also import the components
```

Comment:
```
Generating components lets you pass your generic `FileRouter` and `url` once and
then have typesafe components everywhere, instead of having to pass them
everytime you mount a component, but you can also import the components
```